R/tm_data_table.R

```diff
@@ -365,13 +366,15 @@ srv_data_table <- function(id,
       )
     })
 
-    decorated_data_table_data <-
-      srv_transform_teal_data("decorate", data = data_table_data, transformators = decorators)
+    decorated_data_table_data <- srv_decorate_teal_data(
+      id = "decorator",
+      data = data_table_data,
+      decorators = subset_decorators("table", decorators)
+    )
 
     output$data_table <- DT::renderDataTable(server = server_rendering, {
-      req(data_table_data())
       teal::validate_inputs(iv)
-      decorated_data_table_data()[["table"]]
+      req(decorated_data_table_data())[["table"]]
```

Comment:
`req(decorated_data_table_data())[["table"]]`
or 
` req(decorated_data_table_data()[["table"]])`

or doesn't matter?

Reply:
Doesn't matter, kept it for consistency, but if it's NULL then the result is the same.